frontend/src/stores/external-connections-store.ts

```diff
@@ -19,9 +20,9 @@ export const useExternalConnectionsStore = defineStore('externalConnections', ()
 
   /**
    * Get all external connections for current user
-   * @param {function} call preconfigured API fetch function
+   * @param call preconfigured API fetch function
    */
-  const fetch = async (call) => {
+  const fetch = async (call: FetchExternalConnectionCollection) => {
```

Comment:
Ok, so if I understand correctly, you don't want a type signature at this place for the parameter `call`? Can you give an example of what you mean?

Reply:
I meant to type the request line like:

const reponse: CoolResponse = await call('cool-endpoint');

because technically call doesn't know what its going to return until that point.

Reply:
Ah I see, yes that's a better idea, I can rewrite that 👍🏻 I wanted to do it as parameter type to tell call what we expect to receive, but since that is indeed variable it's better to type the call. Thanks!

Reply:
Added that now!